htmlhint-server/src/server.ts

```diff
@@ -79,10 +80,20 @@ function makeDiagnostic(problem: htmlhint.Error, lines: string[]): server.Diagno
  */
 function getConfiguration(filePath: string): any {
     var options: any;
-    if (settings.htmlhint && settings.htmlhint.options && Object.keys(settings.htmlhint.options).length > 0) {
-        options = settings.htmlhint.options;
-    }
-    else {
+    if (settings.htmlhint) {
+        if (settings.htmlhint.configFile && settings.htmlhint.options && Object.keys(settings.htmlhint.options).length > 0) {
+            throw new Error(`The configuration settings for HTMLHint are invalid. Please specify either 'htmlhint.configFile' or 'htmlhint.options'.`);
```

Comment:
nit:  maybe slightly change wording here: ` The configuration settings for HTMLHint are invalid. Please specify either 'htmlhint.configFile' or 'htmlhint.options', but not both.`

Reply:
I have adjusted the message, thanks.

Reply:
:+1: